builds/e2e/templates/e2e-setup-base-image-update-release.yaml

```diff
@@ -0,0 +1,160 @@
+
+steps:
+- checkout: self
+  clean: true
+  fetchDepth: 100
+  submodules: recursive
+
+- task: AzureKeyVault@1
+  displayName: Get secrets
+  inputs:
+    azureSubscription: $(az.subscription)
+    keyVaultName: $(kv.name)
+    secretsFilter: >-
+      ReleaseContainerRegistryPassword,
+      TestDpsGroupKeySymmetric,
+      TestEventHubCompatibleEndpoint,
+      TestGitHubAccessToken,
+      TestPreviewEventHubCompatibleEndpoint,
+      TestIotedgedPackageRootSigningCert,
+      TestIotHubConnectionString,
+      TestRootCaCertificate,
+      TestRootCaKey,
+      TestRootCaPassword,
+      TestBlobStoreSas
+
+- bash: |
+    echo "Built Images Result is $(builtImages)"
```

Comment:
Is this task necessary? I think you're _always_ using images from the release ACR, _never_ from the artifacts, right? So it seems unecessary to set the `UsePipelineImageArtifacts` variable. And the "Download Current Pipeline Artifacts (downloads everything except Build Images)" task below doesn't need a condition.